packages/strapi-plugin-content-manager/admin/src/components/FilterPickerOption/components.js

```diff
@@ -0,0 +1,122 @@
+import styled, { css } from 'styled-components';
+import Bckg from '../../assets/images/background_input.svg';
+
+const Wrapper = styled.div`
+  min-height: 36px;
+  border-left: ${props => (props.borderLeft ? '3px solid #007EFF' : '0px')};
```

Comment:
`border-left: ${props => (props.borderLeft && '3px solid #007EFF')};`
could be enough ?